gridstatus/ieso.py

```diff
@@ -997,7 +997,8 @@ def _get_latest_resource_adequacy_json(
             )
 
         if last_modified:
-            last_modified = pd.Timestamp(last_modified, tz=self.default_timezone)
+            if last_modified.tz is None:
```

Comment:
if needed, maybe you should raise an error if a tz-naive timestamp is passed. 

if you don't like the raising an error approach and need to pick a timezone, we generally assume tz-naive timestamps are in the the default timezone of the market throughout this library

Reply:
Yeah I think either is fine, I'm doing the latter now albeit to UTC, which I can change to EST (via `self.default_timezone`).